ext/hiera.yaml

```diff
@@ -2,14 +2,8 @@
 :backends:
   - yaml
 :hierarchy:
-  - defaults
-  - "%{clientcert}"
-  - "%{environment}"
-  - global
+  - "nodes/%{clientcert}"
+  - common
 
 :yaml:
-# datadir is empty here, so hiera uses its defaults:
-# - /etc/puppetlabs/code/hieradata on *nix
-# - %CommonAppData%\PuppetLabs\code\hieradata on Windows
-# When specifying a datadir, make sure the directory exists.
-  :datadir:
+  :datadir: "/etc/puppetlabs/code/environments/%{environment}/hieradata"
```

Comment:
I'd state at the very least you need to keep around a comment on how to set it for Windows.